src/Pages/form/Components/UnderLyingDiseaseSection.tsx

```diff
@@ -26,47 +26,90 @@ const PinkCheckbox = withStyles({
   checked: {},
 })((props: CheckboxProps) => <Checkbox color="default" {...props} />);
 
-const checkboxLabel = [
-  "Deny underlying disease",
-  "Hx TIA (same site, within 2 weeks)",
-  "Previous TIA",
-  "Previous stroke",
-  "HT",
-  "DM",
-  "DLP",
-  "Valvular heart disease",
-  "AF",
-  "Coronary heart disease",
-  "CKD",
-  "Peripheral arterial disease",
-  "Obesity",
-  "Smoking",
+const checkboxUnderlying = [
+  { name: "deny", label: "Deny underlying disease" },
```

Comment:
I'm not sure. Which variable is set to formik value? If the answer is name, I think it should be set name that gives meaning (such as "value").